core/src/main/java/io/bisq/core/btc/wallet/WalletsSetup.java

```diff
@@ -300,35 +300,41 @@ private void configPeerNodesForLocalHostBitcoinNode() {
 
     private void configPeerNodes(Socks5Proxy socks5Proxy) {
         List<BitcoinNodes.BtcNode> btcNodeList = new ArrayList<>();
-        walletConfig.setMinBroadcastConnections((int) Math.floor(DEFAULT_CONNECTIONS * 0.8));
         switch (BitcoinNodes.BitcoinNodesOption.values()[preferences.getBitcoinNodesOptionOrdinal()]) {
             case CUSTOM:
-                String bitcoinNodesString = preferences.getBitcoinNodes();
-                if (bitcoinNodesString != null) {
+                final String bitcoinNodesString = preferences.getBitcoinNodes();
```

Comment:
Not sure why the final modifier was added here. Generally, final should be omitted for local variables unless there's some special need (like use within a subsequent lambda expression). It's too noisy otherwise, and IIRC we haven't made a practice of using final on local variables elsewhere in Bisq.

Reply:
Yes was leftover from earlier code where it was used in a lambda.